Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -397,10 +421,15 @@ open class ChatChannelVC: _ViewController,
 
     // MARK: - ChatChannelControllerDelegate
 
+    private var firstMessagesUpdate: Bool = true
     open func channelController(
         _ channelController: ChatChannelController,
         didUpdateMessages changes: [ListChange<ChatMessage>]
     ) {
+        if firstMessagesUpdate {
+            updateUnreadMessagesRelatedComponents()
+            firstMessagesUpdate = false
```

Comment:
Ideally this should not be needed, and we should be using the completion of the synchronize to set the state values for the unread messages related UI, but because the data is not there, we have to do this 🙃

Reply:
ChannelUpdater does not return data, and the observer has not yet received the updates